tests/manage/add_metadata_feature/test_metadata.py

```diff
@@ -61,9 +62,11 @@ def test_metadata_feature_unavailable_for_previous_versions(
         and not suported in previous ODF versions (<4.12) and setmetadata is unavailable,
         for csi-cephfsplugin-provisioner and csi-rbdplugin-provisioner pods
         """
-        external_mode = config.DEPLOYMENT["external_mode"]
-        fs, sc_name = metadata_utils.update_testdata_for_external_modes(
-            sc_name, fs, external_mode=external_mode
+        sc_name = storageclass_name(sc_interface)
+        fs = (
+            ""
+            if sc_interface == constants.OCS_COMPONENTS_MAP["blockpools"]
+            else "fsvol001"
```

Comment:
please save "fsvol001" as a constant

Reply:
"fsvol001" is just any random string not a specific format value , Hence it has uses like that